README.md

```diff
@@ -705,12 +701,12 @@ To build with ccache, use the CMake option **-DBUILD\_WITH\_CCACHE=ON**
 
 ```bash
 $ cd <exiv2dir>
-$ mkdir build ; cd build ; cd build
-$ cmake .. -G "Unix Makefiles" -DBUILD_WITH_CCACHE=ON
-$ cmake --build .
+$ mkdir build
```

Comment:
Remove?

Reply:
Removed in 34fddc8f5d808893b358f829d92b70a9c282ec05